src/interface/meta.thrift

```diff
@@ -60,6 +60,15 @@ union ID {
 }
 
 
+// Geo shape type
+enum GeoShape {
+    ANY = 0,
```

Comment:
What does ANY means?

Reply:
ANY means this property column can store any shape of geo data